LICENSE

```diff
@@ -1,45 +1,3 @@
-Subject to your compliance with applicable law, you are granted the right to use the Licensed Work (defined below) under the terms of the below licenses; provided, however, that if you violate any such applicable law in your use of the Licensed Work, all of your rights and licenses to use (including any rights to reproduce, distribute, install or modify) the Licensed Work will automatically and immediately terminate.
```

Comment:
to confirm, do we want to remove this entire section?

I see that we have things like: `Change License: GNU Affero GPL License v3` below so I am wondering

Reply:
Confirmed - removing the BSL license portionis a clarifying change since the GNU AGPLv3 license is already in effect